### ISSUE TITLE
inconsistent imports

### ISSUE THREAD

**Issue by bboehmke** [Collinux/gohue#14]
I am working on a linux maschine and get some problems with the different import variants.

In the tests the import uses "github.com/collinux/GoHue",
The example uses "github.com/collinux/gohue".

Because directories in linux are case sensetive I get 2 directories in "../src/github.com/collinux/".
This causes an error if I try to run the tests:
`can't load package: package github.com/collinux/GoHue: case-insensitive import collision: "github.com/collinux/GoHue" and "github.com/collinux/gohue"`

Here it would be nice to use the same import for each import.
It is maybe also possible to remove the imports from the test because it is the same package. (Here I am not sure if this is correct because I am new to GO)

Thanks for the libary and best regards

**Comment by Collinux** [comment]
Thank you for pointing this out! 
Fixed in commit 9baef1639cd3efb39b003b76d618a70757a414b4 - changed example code import from "gohue" to "GoHue" to avoid a case-sensitive import collision.

The imports inside the tests are standard practice when writing "_test.go" files since it acts as an external project that imports the package. This provides validation for those who are using the library with the same import line, which in this case helped us catch the import collision. Also examining the tests for functions you're using can help you gain a better understanding of the internals. Maybe another gopher can chime in, I'm sure there are other reasons as well.

**Comment by bboehmke** [comment]
Sorry but I think this test process is not right.

If I try to build the test with from the new path I get:
```
$  cd ~/go/src/github.com/collinux/GoHue
$ go test 
# github.com/collinux/GoHue
import cycle not allowed in test
package github.com/collinux/GoHue (test)
        imports github.com/collinux/GoHue

FAIL    github.com/collinux/GoHue [setup failed]
```
If I understand this correct the error occurs because the test is in the same package as the object itself so the import is not allowed.
Here I found a simular issue https://groups.google.com/forum/#!topic/golang-nuts/3hPEtCNm3XA
